Server/Controls/RegistryTreeView.cs

```diff
@@ -8,11 +8,20 @@ namespace xServer.Controls
 {
     public class RegistryTreeView : TreeView
     {
-
         public RegistryTreeView()
         {
             //Enable double buffering and ignore WM_ERASEBKGND to reduce flicker
             SetStyle(ControlStyles.OptimizedDoubleBuffer | ControlStyles.AllPaintingInWmPaint, true);
         }
+
+        protected override CreateParams CreateParams
+        {
+            get
+            {
+                CreateParams handleParam = base.CreateParams;
+                handleParam.ExStyle |= 0x02000000;// WS_EX_COMPOSITED       
```

Comment:
Sorry about that haha.

The flickering was noticeable without applying this style.  I think it has something to do with the split container control its within, but not entirely sure how it works.